src/components/dialogs/advanced-search.vue

```diff
@@ -0,0 +1,263 @@
+
+            <!-- Validation error message -->
+            <div v-if="invalidSearch" class="error-message copy-small my-n2">
+              Enter at least one search criteria
+            </div>
+
+            <AdvancedSearchForm
+              class="mt-4"
+              :promptSubmit="toggleSearchSubmit"
+              @isInvalid="invalidSearch = $event"
+              @submitForm="submitForm($event)"
+              @closeDialog="emitClose()"
+            />
+          </v-tab-item>
+
+          <!-- Advanced Search Table -->
+          <v-tab-item>
+            <v-card-title>
+              <span>Name Requests Found (18)</span>
+            </v-card-title>
+
+            <v-card-text class="copy-normal pt-5">
+              Select the Name Request you would like to retrieve. To access the detailed summary screen, <br> you will
+              need the email address of phone number of the Applicant.
+            </v-card-text>
+
+            <AdvancedSearchTable
+              class="mt-6"
+              @selectNameRequest="advSearchTab = AdvancedSearchTabs.ADVANCED_SEACH_RETRIEVE"
+            />
+          </v-tab-item>
+
+          <!-- Advanced Search Retrieve Form -->
+          <v-tab-item>
+            <v-card-title>
+              <span>Retrieve Name Request</span>
+            </v-card-title>
+
+            <v-card-text class="copy-normal pt-5">
+              To view the details and status of this Name Request, please enter the <strong>email address or phone
+              number</strong> of the Applicant.
+            </v-card-text>
+
+            <AdvancedSearchRetrieve
+              class="mt-6"
+              :promptSubmit="toggleRetrieveSubmit"
+              @closeDialog="emitClose()"
+            />
+          </v-tab-item>
+
+          <!-- Invalid Results -->
+          <v-tab-item>
+            <v-card-text class="copy-normal results-text">
+              <h3>Too Many Name Requests Found</h3>
+              <span class="my-6">Your search returned more than 1000 results.</span>
+              <span class="mb-4">Please return to the previous screen and narrow your search by selecting more than
+                  one search criteria and/or reducing your date range.</span>
+            </v-card-text>
+          </v-tab-item>
+
+        </v-tabs-items>
+      </v-tabs>
+
+      <v-card-actions id="advanced-search-actions" class="justify-center pa-0">
+          <v-btn
+            id="adv-search-btn-1"
+            class="mr-2"
+            :class="{ 'button-blue': isTabResultsTable || isTabRetrieve }"
+            text
+            outlined
+            @click="advSearchBtn1Actions()"
+          >
+            <span class="px-1">
+              <v-icon v-if="!isTabSearchForm">mdi-chevron-left</v-icon>
+              {{ advSearchBtn1 }}
+              <v-icon v-if="isTabSearchForm">mdi-chevron-right</v-icon>
+            </span>
+          </v-btn>
+          <v-btn
+            id="adv-search-btn-2"
```

Comment:
This button's alignment looks a bit off too, in certain cases:

![image](https://user-images.githubusercontent.com/10002889/117895923-4c3dc880-b274-11eb-9a08-3a5c10b08511.png)

Reply:
ahh yea! The horizontal alignment spacing looks too big on the back btn for sure.

Reply:
This was updated with the icon size adjustment 👍